actions/actions-model.ts

```diff
@@ -0,0 +1,27 @@
+import { database } from "../data/dbConfig";
```

Comment:
that is typescript, we have not learn in lambda, but it are great to see that in here